bc125csv/scanner.py

```diff
@@ -211,7 +206,10 @@ def delete_channel(self, index):
         if channel:
             result = self.send("DCH,%d" % index)
             if not result or result != "DCH,OK":
-                raise ScannerException("Could not delete channel %d." % index)
+                # Fall back to zeroing out the channel
+                result = self.send("CIN,%d,,00000000,,,0,1,0" % index)
+                if not result or result != "CIN,OK":
+                    raise ScannerException("Could not delete channel %d." % index)
```

Comment:
Can you elaborate on why this fallback should be added? The PR description does not mention this addition.